packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -545,6 +545,45 @@ Future<void> buildGradleAar({
     '$successMark Built ${fs.path.relative(repoDirectory.path)}.',
     color: TerminalColor.green,
   );
+  _printHowToConsumeAar(
+    buildMode: androidBuildInfo.buildInfo.modeName,
+    androidPackage: project.manifest.androidPackage,
+    repoPath: repoDirectory.path,
+  );
+}
+
+/// Prints how to consume the AAR from a host app.
+void _printHowToConsumeAar({
+  @required String buildMode,
+  @required String androidPackage,
+  @required String repoPath,
+}) {
+  assert(buildMode != null);
+  assert(androidPackage != null);
+  assert(repoPath != null);
+
+  printStatus('''
+
+${terminal.bolden('Consuming the Module')}
+  1. Open <host>/app/build.gradle
```

Comment:
If this text has to change, for users on stable it may be easier to update the text on the website or on the wiki, than to update the text here. Maybe the status text could have a high-level description of what to do along with a link.

Reply:
I don't expect the instructions to change. Since the status has dynamic values such as paths and package name, I thought it would be easier to copy and paste. 

I agree it's still worth adding a link to it as well. I can follow up with that.